extensions/vscode/src/actions/showAssociateGUID.ts

```diff
@@ -44,12 +44,17 @@ export async function showAssociateGUID(state: PublisherState) {
   }
   await showProgress("Updating Content Record", Views.HomeView, async () => {
     const api = await useApi();
-    await api.contentRecords.patch(
+    const result = await api.contentRecords.patch(
       targetContentRecord.deploymentName,
       targetContentRecord.projectDir,
       {
         guid,
       },
     );
+    if (result.status === 200) {
```

Comment:
Do we need to check the status, or can we surround this in a try catch?

Reply:
Yes we should surround it. Thanks.

Reply:
There is a 204 that can also be returned for the request, which would not be thrown by the Axios status code handler, but that would only occur for a malformed request. I think we're good to switch this to a try/catch, so I've done so.